Contest.Service/Contest.Service/Validators/ParticipantEditDtoValidator.cs

```diff
@@ -0,0 +1,25 @@
+﻿using ContestService.API.DTO.ParticipantDtos;
+using FluentValidation;
+
+namespace ContestService.API.Validators;
+
+public class ParticipantEditDtoValidator : AbstractValidator<ParticipantEditDto>
+{
+    public ParticipantEditDtoValidator()
+    {
+        RuleFor(model => model.Name).NotEmpty().WithMessage("Name is required.")
+            .Length(2, 50).WithMessage("Name must be between 2 and 50 characters.");
+
+        RuleFor(model => model.Surname).NotEmpty().WithMessage("Surname is required.")
+            .Length(2, 50).WithMessage("Surname must be between 2 and 50 characters.");
+
+        RuleFor(model => model.Birthday).Must(birthday =>
```

Comment:
As this logic is repeated, can be extracted to separate method. For example, extension method